install-coverage-pools.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Starting coverage-pools deployment...${LOG_END}"
+
+printf "${LOG_START}Linking dependencies...${LOG_END}"
+
+cd "$WORKDIR/keep-core/solidity"
+yarn link
```

Comment:
I think we use Yarn for the first time in the `local-setup` repo. If so, we need to:
- Mention the need of installing Yarn in [manual setup docs prerequisites](https://github.com/keep-network/local-setup/blob/main/docs/manual-setup.adoc#prerequisites)
- Install Yarn in [auto setup node.js provisioning script](https://github.com/keep-network/local-setup/blob/main/deployments/local-setup-environment/provisioning/install-nodejs.sh)

Reply:
Thanks, I fixed the things you mentioned. I also updated the info about required Go version and updated the `Installing the system` section of documentation.